### DISCUSSION TITLE
Remove Elasticsearch/disable full-text search

### DISCUSSION THREAD

**Discussion by p37307** [mastodon/mastodon#21942] [Q&A] [answered]
I wanted to remove Elasticsearch. I did from my Debian system.  I thought I did but am still seeing "Could not connect to Elasticsearch. Please check that it is running or disable " on the admin Dashboard.

I've searched for a way to disable search. Could someone let me know how I can disable it?

Is it by running tooctl with the command "`tootctl search --disable`" or something else.
Thanks in advance

**Comment by supernovae** [comment, marked as answer]
Modify the configuration in your live/.env.properties 
ES_ENABLED=true

to 

ES_ENABLED=false

And restart web service (or just remove all the ES_ paramters)

**Reply by p37307** [reply]
Thanks. I edited it out previously and restarted, but for some reason, put the edited .env.properties in the Mastodon folder and not the Live.
